File: 3303/3303.cs
public class Solution {
    public int FindTheOccurrenceOfFirstAlmost(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(FindTheOccurrenceOfFirstAlmost(root.left), FindTheOccurrenceOfFirstAlmost(root.right));
    }
}

File: 3303/3303.go
func findTheOccurrenceOfFirstAlmost(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := findTheOccurrenceOfFirstAlmost(root.Left)
	r := findTheOccurrenceOfFirstAlmost(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}